pyaptly/__init__.py

```diff
@@ -679,13 +681,23 @@ def has_dependency(self, dependency):
 
 state = SystemStateReader()
 
+def check_gpg_version():
+    """Check if system gpg point to version 2.x"""
+    global _gpg_bin
+    gpg_version_output, _ = call_output(['gpg', '--version'])
+    gpg_version = gpg_version_output.splitlines()[0].split(" ")[2]
+    if LooseVersion(gpg_version) < LooseVersion("2"):
+        _gpg_bin = '/usr/bin/gpg'
+    else:
+        _gpg_bin = '/usr/bin/gpg1'
```

Comment:
Since we are now checking for gpg we could bail out if there is no appropriate version. Checking again after seelcting the binary and raise a RuntimeException if the version is not < 2

Reply:
would make sense, I update the PR.